packages/react-ai/src/components/AIConversation/types.ts

```diff
@@ -32,7 +32,6 @@ export interface AIConversationInput<
   displayText?: DisplayTextTemplate<AIConversationDisplayText>;
   suggestedPrompts?: SuggestedPrompt[];
   actions?: CustomAction[];
-  showChainOfThought?: boolean;
```

Comment:
Just a type change since we hadn't written the code?

Reply:
This was put up during the scaffold, rest of the code is on a branch but now we can delete both :)